File: backend/src/main/java/com/example/demo/dto/OperatorResponse.java
package com.example.demo.dto;

/**
 * Clase OperatorResponse del sistema de inventario.
 */
public class OperatorResponse {

    private Long id;
    private String username;

    public OperatorResponse(Long id, String username) {
        this.id = id;
        this.username = username;
    }

    public Long getId() { return id; }
    public String getUsername() { return username; }
}

File: AppAndroid/TFGApp/app/src/main/java/com/example/tfgapp/data/model/OperatorResponse.kt
// Archivo comentado: app/src/main/java/com/example/tfgapp/data/model/OperatorResponse.kt
// Explica la responsabilidad de esta clase y los pasos principales del flujo.

package com.example.tfgapp.data.model

// Modelo de datos usado para enviar o recibir información de la API.
data class OperatorResponse(
        val id: Long?,
        val username: String?
) {
        override fun toString(): String {
                return username ?: "Sin nombre"
        }
}
